src/dbus/server/dbus_thread_object.cpp

```diff
@@ -117,6 +120,38 @@ static uint64_t ConvertOpenThreadUint64(const uint8_t *aValue)
     return val;
 }
 
+#if OTBR_ENABLE_TELEMETRY_DATA_API
+static uint32_t TelemetryNodeTypeFromRoleAndLinkMode(const otDeviceRole &aRole, const otLinkModeConfig &aLinkModeCfg)
+{
+    switch (aRole)
+    {
+    case OT_DEVICE_ROLE_DISABLED:
+        return threadnetwork::TelemetryData::NODE_TYPE_DISABLED;
```

Comment:
code style nit: we prefer a single return statement at the end.

Reply:
Done.